Modules/ReportGenerator/Http/Controllers/ReportGeneratorController.php

```diff
@@ -33,14 +33,25 @@ public function index()
      * Show the generated report.
      * @return Response
      */
-    public function getComponents()
+    public function getComponents(Request $request)
     {
-      $option_ids = Input::get($option_ids); // returns an array
+      $option_ids = $request->ids; // returns an array
       // use ids to get the draggable components notes
       // use notes to get the table columns
       // send the data to the view
       //return view('reportgenerator::report')->with('ids', $option_ids);
-      return url('reportgenerator/showReport');
+      $option_ids = serialize($option_ids);
+      return response()->json([
+          'redirecturl' => 'http://localhost:8000/reportgenerator/report/'.$option_ids,
```

Comment:
Shoutcast port? :)

Reply:
Not sure what this affects, but might need to be configurable/discoverable.  For instance, this would not work in my production setup.  localhost does one thing.  127.0.0.1 does something completely different.
-I am sure you have this under consideration, or it is actually a non-issue...I am just making sure you know I am here!  😺

Reply:
I have this @aethelwulffe I'll use the $DB_HOST global.